src/ontology/Makefile

```diff
@@ -39,10 +39,10 @@ SPARQLDIR =                 ../sparql
 COMPONENTSDIR =             components
 ROBOT_PROFILE =             profile.txt
 REPORT_FAIL_ON =            ERROR
-REPORT_LABEL =              
+REPORT_LABEL =              -l true
 REPORT_PROFILE_OPTS =       --profile $(ROBOT_PROFILE)
 OBO_FORMAT_OPTIONS =        
-SPARQL_VALIDATION_CHECKS =  owldef-self-reference iri-range label-with-iri multiple-replaced_by 
```

Comment:
not sure why these got thrown away, were these like a default profile?

Reply:
Its because you decided to customise the list - these were the defaults when you do not customise the list of checks!

Reply:
hmm okay so I will just re-capitulate the list here into the "custom" list so they don't get thrown away.